DeepCat.js

```diff
@@ -166,6 +176,8 @@
 			newSearchTerms[userParameters['searchTermNum']] = newSearchTermString;
 		}
 
+		errors.concat(errorMessages);
```

Comment:
You should use ``error`` in ``computeErrors`` and call ``addErrorMsgField( error )`` when both computing methods are finished.